src/index.html

```diff
@@ -6,13 +6,58 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__img">
+        <img
+          class="card__item"
+          src="./images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `img` tag should be a self-closing tag. Please update it accordingly.